frontend/src/pages/projects/screens/detail/overview/configuration/ConfigurationCard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  FlexItem,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import { ProjectObjectType, SectionType } from '~/pages/projects/types';
+import { ProjectDetailsContext } from '~/pages/projects/ProjectDetailsContext';
+import HeaderIcon from '~/pages/projects/components/HeaderIcon';
+
+interface ConfigurationCardProps {
+  objectType: ProjectObjectType;
+  title: string;
+  description: string;
+  navSection: string;
+}
+const ConfigurationCard: React.FC<ConfigurationCardProps> = ({
+  objectType,
+  title,
+  description,
+  navSection,
+}) => {
+  const navigate = useNavigate();
+  const { currentProject } = React.useContext(ProjectDetailsContext);
+
+  return (
+    <Card className={css('odh-overview-card', SectionType.organize)} style={{ height: '100%' }}>
```

Comment:
`odh-overview-card` related css file isn't imported. Also it's a class that's tied to the `OverviewCard` component. Perhaps make a reusable component to encapsulate this behavior.